### DISCUSSION TITLE
Live spell check not working in 4.0

### DISCUSSION THREAD

**Discussion by sasamavros** [SubtitleEdit/subtitleedit#7378] [Q&A]
There seems to be a problem in live spell check in Subtitle Edit 4.0. Can it be solved?

**Comment by niksedk** [comment]
Should be fixed in latest beta: https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.0/SubtitleEditBeta.zip

**Reply by sasamavros** [reply]
no luck.

**Reply by AndyBrizhak** [reply]
Same problem on  portable version
